src/App.vue

```diff
@@ -689,6 +691,8 @@ export default class App extends Mixins(
         resources = RegistrationResources
         parseFiling = this.parseRegistrationDraft
         break
+      default:
+        throw new Error(`Invalid filing type = ${this.getFilingType}`)
```

Comment:
Sometimes, during development, an update event occurred but filing type was cleared. Without this check, the code would try to parse "nothing" and display an error. This catches the error as early as possible.